domain/src/main/java/org/sagebionetworks/research/domain/result/interfaces/Result.java

```diff
@@ -45,7 +43,7 @@
     /**
      * @return The time this result ended.
      */
-    @Nullable
+    @NonNull
```

Comment:
Shouldn't this be Nullable since the task result get's used while the task is running?

Reply:
Yes, and I agree putting in a placeholder was hacky.
I've put this back to @Nullable and put @NonNull on the children interfaces (you can do that but you can't override a @NonNull with a @Nullable)